zebra-chain/src/commitments/sapling.rs

```diff
@@ -0,0 +1,341 @@
+//! Sapling note and value commitments and types.
+
+#![allow(clippy::unit_arg)]
```

Comment:
What false-positive does this avoid? Looking at the lint it seems like it probably indicates a bug, so I'm wondering if I'm missing something

Reply:
Whatever it was, it's not there anymore, removing
```suggestion
```